Gemfile

```diff
@@ -19,6 +19,8 @@ gem 'devise'
 
 gem 'pronto', '~> 0.9.5'
 gem 'pronto-rubocop', '~> 0.9.0'
+gem 'awesome_print', '~> 1.8'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `awesome_print` should appear before `pronto-rubocop`.